### PR TITLE
[PLACEHOLDER UNTIL HEDIFF] [DNM] Wounds

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request

You should never merge this
Do not merge
dnm

Port of https://github.com/Whitesands13/Whitesands/pull/886
In addition to the PRs listed above...
https://github.com/tgstation/tgstation/pull/56056
https://github.com/tgstation/tgstation/pull/56599
https://github.com/tgstation/tgstation/pull/56637 (this was was reverted at the moment)
https://github.com/tgstation/tgstation/pull/56625
https://github.com/tgstation/tgstation/pull/56697
https://github.com/tgstation/tgstation/pull/57180
https://github.com/tgstation/tgstation/pull/58006
https://github.com/tgstation/tgstation/pull/58682
https://github.com/tgstation/tgstation/pull/59561
https://github.com/tgstation/tgstation/pull/59816
https://github.com/tgstation/tgstation/pull/59848
https://github.com/tgstation/tgstation/pull/61196
https://github.com/tgstation/tgstation/pull/63140
https://github.com/tgstation/tgstation/pull/63168
https://github.com/hrzntal/horizon/pull/27

There is a bug where disabled legs do not get disabled

## Why It's Good For The Game

Better medical gameplay than do bone repair surgery every so often 

## Changelog
:cl:
add: ports tg wounds
/:cl:

### CODE REVIEW

**Comment by Nobelium-XVIII** [comment]
When's datum-med

**Comment by rye-rice** [comment]
datum-deez-nuts

**Comment by Ms-Mee** [comment]
When the wounds are sus

**Comment by rye-rice** [comment]
oops

File: code/datums/mood_events/generic_negative_events.dm [hunk 1]
```diff
@@ -91,20 +91,23 @@
 	mood_change = -2
 	timeout = 2 MINUTES
 
-/* WS - For shame /TG/... for shame.
```
Comment (Ms-Mee) [inline code/datums/mood_events/generic_negative_events.dm:94]: Oh no it enables the tabling moodlet

Comment (rye-rice) [reply code/datums/mood_events/generic_negative_events.dm:94]: WHAT

Comment (Ms-Mee) [reply code/datums/mood_events/generic_negative_events.dm:94]: You heard me bud

File: code/game/objects/items/dualsaber.dm [hunk 2]
```diff
@@ -20,9 +20,12 @@
 	light_color = LIGHT_COLOR_ELECTRIC_GREEN
 	light_on = FALSE
 	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "tore", "lacerated", "ripped", "diced", "cut")
+	block_chance = 75
```
Comment (Ms-Mee) [inline code/game/objects/items/dualsaber.dm]: Kinda sussy

File: code/modules/jobs/job_types/virologist.dm [hunk 3]
```diff
@@ -11,6 +11,9 @@
 	wiki_page = "Infections" //WS Edit - Wikilinks/Warning
 	special_notice = "Don't release viruses, harmful or not, without the CMO's permission." //WS Edit - Wikilinks/Warning
 
+	skills = list(/datum/skill/healing = SKILL_EXP_NOVICE)
```
Comment (Ms-Mee) [inline code/modules/jobs/job_types/virologist.dm]: This PR has so much stuff that's not even really that related to wounds holy shit
Also a bunch of one and two-liners that just change sharpness and similar

Comment (rye-rice) [reply code/modules/jobs/job_types/virologist.dm]: I really need to gut out the skills thing that was added
Seems like the last ws coders thought making wound sounds only heard from people with novice in medical skills was a good idea

File: code/modules/surgery/bodyparts/_bodyparts.dm [hunk 4]
```diff
@@ -20,17 +20,13 @@
 	var/use_digitigrade = NOT_DIGITIGRADE //Used for alternate legs, useless elsewhere
 	var/list/embedded_objects = list()
 	var/held_index = 0 //are we a hand? if so, which one!
-	var/render_like_organic = FALSE // TRUE is for when you want a BODYPART_ROBOTIC to pretend to be a BODYPART_ORGANIC.
+	var/render_like_organic = FALSE	// TRUE is for when you want a BODYPART_ROBOTIC to pretend to be a BODYPART_ORGANIC.	//WS edit - IPCs
```
Comment (Ms-Mee) [inline code/modules/surgery/bodyparts/_bodyparts.dm]: True insanity, comment inside a comment that isn't even actually correct

**Review by Ms-Mee — COMMENTED** [review]
Alrighty, looked through most of it and I'll definetly need to test locally as to what exactly is making dismemberment not happen, it could be it's not trying to call even the check, failing the check, or something else entirely, one thing is that it makes dismemberment need both bones and flesh mangled which
Ehhh doesn't seem quite right for all cases unless I missed something

File: code/modules/jobs/job_types/virologist.dm (hunk 3 above)
Comment (rye-rice) [reply code/modules/jobs/job_types/virologist.dm]: job skill system was removed

File: code/game/objects/items/dualsaber.dm (hunk 2 above)
Comment (rye-rice) [reply code/game/objects/items/dualsaber.dm]: this was removed

File: code/datums/mood_events/generic_negative_events.dm (hunk 1 above)
Comment (rye-rice) [reply code/datums/mood_events/generic_negative_events.dm:94]: It is sealed again

File: code/__DEFINES/DNA.dm [hunk 5]
```diff
@@ -102,12 +102,12 @@
...
-#define TR_KEEPORGANS (1<<8)
-#define TR_KEEPSTUNS (1<<9)
-#define TR_KEEPREAGENTS (1<<10)
-
+#define TR_KEEPSE		(1<<5)
+#define TR_DEFAULTMSG	(1<<6)
+#define TR_KEEPORGANS	(1<<8)
+#define TR_KEEPSTUNS	(1<<9)
+#define TR_KEEPREAGENTS	(1<<10)
+#define TR_KEEPSTAMINADAMAGE (1<<11)
```
Comment (ZephyrTFA) [inline code/__DEFINES/DNA.dm]: Why are you changing the spacing of the defines?

Comment (rye-rice) [reply code/__DEFINES/DNA.dm]: This was a mistake from trying to port the original pr

**Comment by rye-rice** [comment]
Once again this is ready, just needs testing

**Comment by rye-rice** [comment]
conflicts fixed

**Comment by rye-rice** [comment]
See https://github.com/retlaw34/Shiptest/tree/wounds_kapu_betterlimbs for my attempt at fixing this PR's merge conflicts
Screenshot_2107

**Pull request closed without merging**